### PR TITLE
Deploying on Google App Engine  번역

### PR DESCRIPTION
**Pull request by SeoEunkyo**
검토 부탁 드립니다.

### CODE REVIEW

File: fastai-env/Deploying on Google App Engine.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
+﻿# Google App Engine에서 배포 하기
+
+## 목차
+
+-  [배포를 위한 start pack 가져오기](#downloadstarerpack)
+-  [프로젝트 별 설정](#per-project_setup)
```
Comment (kkweon) [inline fastai-env/Deploying on Google App Engine.md]:
```suggestion
-  [프로젝트 설정](#per-project_setup)
```

i think it sounds more natural without the translation of `per`

File: fastai-env/Deploying on Google App Engine.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+## 배포를 위한 start pack 가져오기<span id="downloadstarerpack"></span>
+```
+wget https://github.com/fastai/course-v3/raw/master/docs/production/google-app-engine.zip
+
+unzip google-app-engine.zip
+
+cd google-app-engine/app
+```
+
+## 프로젝트 별 설정<span id="per-project_setup"></span>
```
Comment (kkweon) [inline fastai-env/Deploying on Google App Engine.md]:
```suggestion
## 프로젝트 설정<span id="per-project_setup"></span>
```

File: fastai-env/Deploying on Google App Engine.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+**학습된 모델 업로드** 
+
+  Google Drive 또는 Dropbox와 같은 클라우드 서비스에 학습된 모델 파일을 업로드(예:  stage-2.pth) 하세요. 그리고 파일의 다운로드 링크를 복사하십시오.
+**주의:**  다운로드 링크는 직접 파일이 다운로드를 시작해야합니다. 일반적으로 사용되는 뷰를 제공하는 공유링크와는 다른것 입니다.  필요시 아래의 주소에서 링크를 변경 할 수 있습니다.
```
Comment (kkweon) [inline fastai-env/Deploying on Google App Engine.md]: This doesn't sound very natural. 
Especially,

> 다운로드 링크는 직접 파일이 다운로드를 시작해야합니다

Can you take a look again?

File: fastai-env/Deploying on Google App Engine.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+(공유링크 -> 직접 다운로드 링크로 변경  : https://rawdownload.now.sh/)
+만약  배포 테스트만을 원한다면, Jeremy’s 곰 분류 모델이 기본적으로 설정 된 모델을 그래로 사용하면 됩니다.  모델의 가중치 URL은 이미 샘플 App에 기입 되어 있으므로 이 단계를 건너뛸 수 있습니다.
```
Comment (kkweon) [inline fastai-env/Deploying on Google App Engine.md]:
```suggestion
만약  배포 테스트만을 원한다면, Jeremy’s 곰 분류 모델이 기본적으로 설정 된 모델을 그[DAE]로 사용하면 됩니다. 모델의 가중치 URL은 이미 샘플 App에 기입 되어 있으므로 이 단계를 건너뛸 수 있습니다.
```

Typo

File: fastai-env/Deploying on Google App Engine.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+
+
+<br>
+
+**App을 나의 모델로 변경하기** 
+ 
+ 
+  app 폴더에 있는 ``server.py``를 편집기를 통하여 열고 ``model_file_url`` 이 부분을 위에서 복사 해둔 url로 변경해주세요.   같은 파일에서 ``classes = ['black', 'grizzly', 'teddys'] `` 에 부분 클래스 내용을 모델에서 예상하는 것으로 변경하세요.
```
Comment (kkweon) [inline fastai-env/Deploying on Google App Engine.md]:
```suggestion
  app 폴더에 있는 `server.py`를 편집기를 통하여 열고 `model_file_url` 이 부분을 위에서 복사 해둔 url로 변경해주세요.   같은 파일에서 `classes = ['black', 'grizzly', 'teddys'] ` 에 부분 클래스 내용을 모델에서 예상하는 것으로 변경하세요.
```

File: fastai-env/Deploying on Google App Engine.md [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+
+
+
+## 변경한 App GitHub로 업로드<span id="uploadapp"></span>
+
+변경 완료한 App의 폴더를 GitHub로 Push하고 repo의 URL을 복사 합니다.
+
+## 배포<span id="deploy"></span>
+우선 Google Cloud 대쉬보드를 열어주세요, 그리고 **_Create Project_** 버튼을 클릭한 다음 새 GCP 프로젝트의 이름을 정해 줍니다.  새로운 비용청구를 위한 계정을 생성하거나 기존의 청구 계정을 이용 하여 설정 가능 합니다.  화면은 아래와 같을 것 입니다.
```
Comment (kkweon) [inline fastai-env/Deploying on Google App Engine.md]:
```suggestion
우선 Google Cloud 대쉬보드를 열어주세요, 그리고 **_Create Project_** 버튼을 클릭한 다음 새 GCP 프로젝트의 이름을 정해 줍니다. 새로운 비용청구를 위한 계정을 생성하거나 기존의 청구 계정을 이용 하여 설정 가능 합니다. 화면은 아래와 같을 것 입니다. 
```

File: fastai-env/Deploying_on_Google_App_Engine.md [hunk 7]
```diff
@@ -0,0 +1,90 @@
+﻿# Google App Engine에서 배포 하기
```
Comment (kkweon) [inline fastai-env/Deploying_on_Google_App_Engine.md]: Please remove this file or the other.

**Review by kkweon — CHANGES_REQUESTED** [review]
Good work. First pass review done.

File: fastai-env/Deploying_on_Google_App_Engine.md (hunk 7 above)
Comment (deep-diver) [reply fastai-env/Deploying_on_Google_App_Engine.md]: I have removed this one

File: fastai-env/Deploying on Google App Engine.md (hunk 3 above)
Comment (deep-diver) [reply fastai-env/Deploying on Google App Engine.md]: @kkweon 

looks like he has made some changes, but it was not done through this in-line code editor.

File: fastai-env/Deploying on Google App Engine.md (hunk 2 above)
Comment (kkweon) [reply fastai-env/Deploying on Google App Engine.md]: I still don't see the change.

**Comment by deep-diver** [comment]
@kkweon all the required changes are applied, could you please check? Thank you

**Review by kkweon — APPROVED** [review]

**Pull request merged**